webapp/src/components/App/Tasks/JobTableView.tsx

```diff
@@ -67,7 +69,11 @@ function JobTableView(props: Props) {
   const { content, refresh } = props;
   const [t] = useTranslation();
   const [dateOrder, setDateOrder] = useState<NonNullable<TableSortLabelProps["direction"]>>("desc");
+  const [userOrder, setUserOrder] = useState<NonNullable<TableSortLabelProps["direction"]>>("desc");
   const [filterType, setFilterType] = useState<FilterListType | "">("");
+  const [userFilterValue, setUserFilterValue] = useState<string>(
```

Comment:
`filterUser` by consistency with the others state filters.

And you cane remove `<string>`